drag_and_drop_v2/drag_and_drop_v2.py

```diff
@@ -50,6 +50,26 @@ class DragAndDropBlock(XBlock):
         default=1
     )
 
+    item_background_color = String(
+        display_name="Item background color",
+        help=(
+            "Background color to use for draggable items. "
+            "Defaults to color specified by current theme, or blue if no theme is set."
+        ),
+        scope=Scope.settings,
+        default=""
```

Comment:
Nit: Always include a trailing comma in objects and lists that have one key-value per line, to make future diffs nicer.